cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md

```diff
@@ -18,17 +18,17 @@ runtime-spec-compatible `dest/config.json`.
 **--help**
   Print usage statement
 
-**--ref**
+**--ref**=""
   The ref pointing to the manifest of the OCI image. This must be present in the "refs" subdirectory of the image. (default "v1.0")
 
-**--rootfs**
+**--rootfs**=""
   A directory representing the root filesystem of the container in the OCI runtime bundle. It is strongly recommended to keep the default value. (default "rootfs")
 
-**--type**
+**--type**=""
   Type of the file to unpack. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "imageLayout,image"
 
-**--version**
-  Print version information and exit.
+**-v**, **--version**=*true*|*false*
+  Print version information and exit. Default is false.
```

Comment:
This is the optional parameters of the two values, I think there can be here. This allows the user to better understand this parameter.